coalib/misc/Exceptions.py

```diff
@@ -20,10 +15,10 @@ def get_exitcode(exception, log_printer=None):
         exitcode = exception.code
     elif isinstance(exception, VersionConflict):
         log_message = Constants.VERSION_CONFLICT_MESSAGE % str(exception.req)
-        log_printer.log_exception(log_message, exception)
+        LogPrinterMixin().log_exception(log_message, exception)
```

Comment:
Its not just about format.
Current `LogPrinterMixin.log_exception` logs two items:

1. `log_message` received as parameter and its level is also received as parameter with default value as `logging.ERROR`.
2. Traceback of the exception. Exception is received as parameter.

That's why I created a util which provides similar functionality. Let me know if you've better alternative.